### DISCUSSION TITLE
How to create schedule only for one execution?

### DISCUSSION THREAD

**Discussion by vadympop** [agronholm/apscheduler#849] [Q&A]
I need to create the schedule only for one execution, something like delayed function. How I can do this?

**Comment by agronholm** [comment]
Are you sure you've read the documentation? That's what [DateTrigger](https://apscheduler.readthedocs.io/en/3.x/userguide.html#choosing-the-right-scheduler-job-store-s-executor-s-and-trigger-s) is for.
